token/js/src/actions/createAssociatedTokenAccount.ts

```diff
@@ -24,7 +24,8 @@ export async function createAssociatedTokenAccount(
     owner: PublicKey,
     confirmOptions?: ConfirmOptions,
     programId = TOKEN_PROGRAM_ID,
-    associatedTokenProgramId = ASSOCIATED_TOKEN_PROGRAM_ID
+    associatedTokenProgramId = ASSOCIATED_TOKEN_PROGRAM_ID,
+    idemPotent = false,
```

Comment:
Same here -- let's not have this parameter and create a separate function `createAssociatedTokenAccountIdempotent`.